src/ui/utils/app.ts

```diff
@@ -0,0 +1,30 @@
+import { TimeStampedPointRange } from "@recordreplay/protocol";
+export type TimeRange = {
+  start: number;
+  end: number;
+};
+
+export function getNonLoadingRegionTimeRanges(
+  loadingRegions: TimeStampedPointRange[],
+  endTime: number
+): TimeRange[] {
+  if (!loadingRegions.length) {
+    return [{ end: endTime, start: 0 }];
```

Comment:
Why do we use `startTime`/`endTime` for some things and `start`/`end` for other things? At least we have TypeScript to help but it's a little confusing :)

It'a also interesting that sometimes `end` refers to the number but other times its an object with a `.time` property 😁

Reply:
Same comment here: https://github.com/RecordReplay/devtools/pull/6477#discussion_r854737015

There's a nice chunk of work that can simplify this once we get around to cleaning up this function, especially [these calls](https://github.com/RecordReplay/devtools/blob/8a1cbf7fa800224d922bfea230a957a0f1669a97/src/ui/actions/timeline.ts#L96-L99). But since this is affecting users I stopped short of doing that bit for now.